assets/thanos-querier/prometheus-rule.yaml

```diff
@@ -23,9 +23,9 @@ spec:
         /
           sum by (job) (rate(http_requests_total{job="thanos-querier", handler="query"}[5m]))
         ) * 100 > 5
-      for: 1h
+      for: 5m
       labels:
-        severity: warning
+        severity: critical
```

Comment:
We want to keep these alerts to `warning` severity and keep the `for` clause to `1h`.